containers/kubernetes/how-to/revoke-kubeconfig.mdx

```diff
@@ -0,0 +1,71 @@
+---
+meta:
+  title: How to revoke a kubeconfig of a Kapsule cluster
+  description: This page explains how to revoke a kubeconfig of a Kapsule cluster
+content:
+  h1: How to revoke a kubeconfig of a Kapsule cluster
+  paragraph: This page explains how to revoke a kubeconfig of a Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster kubeconfig revoke iam
+dates:
+  validation: 2023-11-28
+  posted: 2023-11-28
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+Kubernetes Kapsule uses now an updated authentication system, based on IAM tokens. **This leads to the simultaneous existence of two authentication mechanisms.**
```

Comment:
```suggestion
Kubernetes Kapsule now uses an updated authentication system, based on IAM tokens. **This leads to the simultaneous existence of two authentication mechanisms.**
```

Reply:
I think it would be nice to link to the IAM doc more on this page, eg to the concepts  mentioned like IAM group, policy, principal etc.